components/playlist/browser/playlist_tab_helper_observer.h

```diff
@@ -17,11 +17,11 @@ class PlaylistTabHelperObserver : public base::CheckedObserver {
  public:
   virtual void PlaylistTabHelperWillBeDestroyed() = 0;
```

Comment:
optional: How about giving empty body to this function for consistency?

Reply:
The intent here was to make sure that observers remove themselves prior to tab helper destruction, hence the pure specifier.